### ISSUE TITLE
Can we have only one script and adjust the behavior based on some param/env var? Or distribute egg?

### ISSUE THREAD

**Issue by andywaltlova** [oamg/convert2rhel-worker-scripts#48]
It's hard to maintain two almost same scripts - from testing POV it doesn't make sense, our work would be much simpler if we had just one script and either call convert or analyze based on some input param /env var.

Other solution could be to distribute whole egg, that way we could split the scripts to multiple files.

**Comment by r0x0d** [comment]
Yes, that would be the ideal. We have a lot of technical debt right now with this repository (and our setup with vagrant). I think we should tackle this first before trying to work on the other issues.
